rest_api/controller/search.py

```diff
@@ -195,3 +219,46 @@ def faq_qa(model_id: int, request: Question):
     logger.info(json.dumps({"request": request.dict(), "results": results}))
 
     return {"results": results}
+
+
+# TODO: Need to convert response object as well?
```

Comment:
I think it'd be nice to have the response object cast to the `Answers` schema so we can validate the response format.